code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -411,6 +411,16 @@
 	tastes = list("cherry" = 1, "crepe" = 1)
 	foodtype = GRAIN | FRUIT | SUGAR
 
+/obj/item/reagent_containers/food/snacks/englishbreakfast
+	name = "full english breakfast"
+	desc = "The maximum amount of breakfasting. Anymore and you reach hubris."
+	icon_state = "englishbreakfast"
+	bonus_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/nutriment/vitamin = 1)
+	list_reagents = list(/datum/reagent/consumable/nutriment = 10, /datum/reagent/consumable/nutriment/vitamin = 2)
+	tastes = list("a symphony of breakfast foods" = 3)
```

Comment:
this should be broken into its constituent parts probably

Reply:
see above